### ISSUE TITLE
Black Screen with Tales of Eternia (Europe)

### ISSUE THREAD

**Issue by BrajikHekrsna** [OpenEmu/OpenEmu#2627]
I've been trying to play Tales of Eternia for PSP using OpenEmu but whenever I open it, there's just a black screen and then it won't close and I have to force quit. It should be compatible with PPSSPP, as far as I know. 
I'm running OS X 10.11.5 and OpenEmu v2.0.2.

Edit: I tried loading another game but ran into the same issue.

**Comment by mrvacbob** [comment]
Is anything special printed in Console when this happens?

We should try to be resistant to game cores freezing up.

**Comment by clobber** [comment]
>It should be compatible with PPSSPP, as far as I know. 

I didn't run this yet, but it looks like a regression in PPSSPP 1.2.2.

**Comment by clobber** [comment]
>7/6/16 5:30:50.185 PM OpenEmuHelperApp[59187]: ERROR: Failed to identify file

Never saw that one before. Bad dump of the game?

**Comment by clobber** [comment]
Yeah, you loaded a bad dump of the game. Probably TRSI-TOE.ISO (Tales_of_Eternia_GERMAN_PSP_EUR-TRSI) -- the JPN release loads fine.

**Comment by leomeloxp** [comment]
Hi,

Not sure if mods rather have a new issue or re-open this one but I've also got this issue with the same game and same error, running OpenEmu 2.0.3 and PPSSPP core 1.2.2.1. Logs:

```
OpenEmu[62768]: SETUP DONE.
OpenEmuHelperApp[62771]: Rate change 0.000000 -> 1.000000
OpenEmuHelperApp[62771]: ERROR: Failed to identify file
coreaudiod[189]: SecTaskLoadEntitlements failed error=22
OpenEmu[62768]: Display's nominal refresh rate is 59.99 Hz
launchservicesd[81]: SecTaskLoadEntitlements failed error=22
```

I would just like to point out that I also took the time to install PPSSPP on my mac as a standalone and if I run the same ISO from there it works just fine. I also understand you suggested using a different ISO dump but I've already played this game on my PSP (now dead) and am seeking to import my save to the mac (which I could do with PPSSPP but OpenEmu is just way too good not to use it).

Happy to help with debugging this further in any way I can.

**Comment by clobber** [comment]
>ERROR: Failed to identify file

^----- that is an error message from PPSSPP.

This issue is still the same: PPSSPP 1.2.2 cannot identify that ROM. Not an OpenEmu problem.

If you try to load the game with PPSSPP 1.2.2 standalone (which there doesn't seem to be any precompiled builds of) and not a 'nightly', you should get the same result.

**Comment by leomeloxp** [comment]
My standalone PPSSPP version is 1.2.2-875-g028e98a (which I got from their site today) so maybe it has recently been fixed?

Is there a way I can manually import my standalone PPSSPP as the Core for PSP on OpenEmu? If not, I guess I carry on playing on standalone until the update reaches OpenEmu's core.

Cheers for the fast response btw, appreciate it.

**Comment by clobber** [comment]
>Is there a way I can manually import my standalone PPSSPP as the Core for PSP on OpenEmu? 

Nope.

**Comment by clobber** [comment]
The Europe dump of the game appears to once again load with PPSSPP 1.3. We will have a core plugin update out soon.

**Comment by clobber** [comment]
Sorry, fixed here for real https://github.com/OpenEmu/OpenEmu/commit/767aa03c90c136257bde267302e4e98fe2198b8d

This will have to come in the next OpenEmu update.

**Comment by AlexKornitzer** [comment]
I have this error too for Burnout Legends (E), it runs fine in the latest `Dev Working` build of PPSSPP but fails in OpenEMU.

**Comment by clobber** [comment]
@AlexKornitzer See [my comment](https://github.com/OpenEmu/OpenEmu/issues/2627#issuecomment-251025521) before yours...

**Comment by AlexKornitzer** [comment]
@clobber ah my bad thought you meant it was fixed in the 1.3 core update, ooops :)
